### PR TITLE
Fix specs

### PR DESCRIPTION
**Pull request by raphink**
This PR does the following:
- Fix specs for Rspec2
- Add missing dependencies to gemspec
- Add a Gemfile
- Plug into Travis CI
- Fix UTF-8 related bug in cucumber tests

See builds on https://travis-ci.org/raphink/ruby_gpg/builds

### CODE REVIEW

File: ruby_gpg.gemspec [hunk 1]
```diff
@@ -49,26 +49,11 @@ Gem::Specification.new do |s|
...
-    s.add_dependency(%q<yard>, [">= 0"])
-    s.add_dependency(%q<cucumber>, [">= 0"])
-    s.add_dependency(%q<mocha>, [">= 0.9.8"])
-  end
+  s.add_development_dependency(%q<rake>, ["~> 10.0"])
+  s.add_development_dependency(%q<rspec>, ["~> 2.99.0"])
+  s.add_development_dependency(%q<yard>, [">= 0"])
+  s.add_development_dependency(%q<cucumber>, [">= 0"])
+  s.add_development_dependency(%q<mocha>, [">= 1.1.0"])
+  s.add_development_dependency(%q<jeweler>, [">= 2.0.0"])
```
Comment (raphink) [inline ruby_gpg.gemspec:57]: @blaix I simplified this block because this seems to target _ancient_ environments.

File: spec/ruby_gpg_spec.rb [hunk 2]
```diff
@@ -4,9 +4,12 @@
...
       case part_of_command
-      when Regexp: command =~ part_of_command
-      when String: command.include?(part_of_command)
-      else raise "Can't match that"
+      when Regexp
+        command =~ part_of_command
+      when String
+        command.include?(part_of_command)
+      else
+        raise "Can't match that"
```
Comment (raphink) [inline spec/ruby_gpg_spec.rb:12]: @blailx I could not get the syntax with colons to work. I don't know if it's an old syntax that is now deprecated…

File: features/step_definitions/ruby_gpg_steps.rb [hunk 3]
```diff
@@ -20,7 +20,7 @@
 end
 
 Given /^the string should not be "([^\"]*)"$/ do |string|
-  @string.strip.should_not == string.strip
+  @string.encode('UTF-8', 'binary', invalid: :replace, undef: :replace, replace: '').strip.should_not == string.strip
```
Comment (raphink) [inline features/step_definitions/ruby_gpg_steps.rb:23]: @blaix UTF-8 in Ruby 1.9+

**Comment by raphink** [comment]
@blaix have you had time to review this?

**Comment by blaix** [comment]
Not yet. Sorry.

**Comment by blaix** [comment]
This is awesome! Thanks.

**Pull request merged**